### DISCUSSION TITLE
Copilot not working on JetBrains Rider

### DISCUSSION THREAD

**Discussion by Oyshoboy** [community/community#6950] [Copilot Conversations] [answered]
Hey! It's seems like the plugin for Rider is out, at least it's become available in a marketplace. After installation nothing happening, and there are no any authorization windows or copilot icons. 

Anybody knows if I can call for the authorization window manually?

**Comment by Oyshoboy** [comment, marked as answer]
Found this window and now everything is working!

1. Double shift 
2. Search for copilot
3. Press "Github Login"

![image](https://user-images.githubusercontent.com/23486183/139337601-46c59e26-b14e-40e3-add0-5ce3f08ba033.png)

**Reply by jansorg** [reply]
There's also the documentation at https://github.com/github/copilot-docs/blob/main/docs/jetbrains/README.md

**Reply by Corical** [reply]
Link broken, updated link https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-a-jetbrains-ide
